src/components/mgt-login/mgt-login.scss

```diff
@@ -12,14 +12,17 @@
 mgt-login {
   --font-size: #{$ms-font-size-m};
   --font-weight: #{$ms-font-weight-semibold};
+  --width: '100%';
   --height: '100%';
   --margin: 0;
   --padding: 12px 20px;
   --color: #{$ms-color-neutralDark};
   --background-color: transparent;
   --background-color--hover: #{$ms-color-neutralLight};
-  --popup-content-background-color: white;
-  --popup-command-font-size: #{$ms-font-size-s};
+  --menu-content-background-color: white;
+  --menu-command-font-size: #{$ms-font-size-s};
```

Comment:
breaking change!

Also, docs need to be updated

Reply:
Right, sorry.  Thoughts on name change?  I thought menu was clearer, but happy to revert.

Reply:
I'll leave it for now, we can change later if needed.